lib/logging/CLogger.cpp

```diff
@@ -387,7 +389,8 @@ void CLogConsoleTarget::setColorMapping(const CColorMapping & colorMapping) { th
 CLogFileTarget::CLogFileTarget(boost::filesystem::path filePath, bool append)
 	: file(std::move(filePath), append ? std::ios_base::app : std::ios_base::out)
 {
-	formatter.setPattern("%d %l %n [%t] - %m");
+//	formatter.setPattern("%d %l %n [%t] - %m");
```

Comment:
why this change?

Reply:
VCMI write so much logs, so date formatting in too slow